docs/tutorials/02_ui/viz_tree_ui.py

```diff
@@ -0,0 +1,74 @@
+from fury import ui, window
+from fury.ui.core import Disk2D, Rectangle2D
+from fury.ui.elements import LineSlider2D, ListBox2D
+from fury.ui.containers import ImageContainer2D
+
+structure = [{'Containers': ['Panels', 'ImageContainers']},
+             {'Elements': ['ListBox', 'LineSlider']},
+             {'Core': ['Rectangle', 'Disk']}]
+
+tree = ui.elements.Tree2D(structure=structure, tree_name="FURY UI Breakdown",
+                          size=(500, 500), position=(0, 0),
```

Comment:
you should change the position of the UI. When you start the tutorial, it is strange to see it hidden

Reply:
Done.